client/apollo/react/src/TimelineVertical/TimelineVerticalCommon.tsx

```diff
@@ -0,0 +1,27 @@
+import { PropsWithChildren, ReactNode } from "react";
+import classNames from "classnames";
+
+export type TimelineVerticalProps = PropsWithChildren<{
+  title: string;
+  tag: ReactNode;
+  className?: string;
+}>;
+
+export const TimelineVertical = ({
+  title,
+  children,
+  tag,
+  className,
+}: TimelineVerticalProps) => {
+  return (
+    <section className={classNames("af-timeline-vertical", className)}>
+      <header className="af-timeline-vertical__header">
+        {tag}
+        <h4 className="af-timeline-vertical__title">{title}</h4>
+      </header>
+      {Boolean(children) && (
+        <main className="af-timeline-vertical__description">{children}</main>
+      )}
+    </section>
+  );
+};
```

Comment:
Pas besoin de return

```suggestion
export const TimelineVertical = ({
  title,
  children,
  tag,
  className,
}: TimelineVerticalProps) => (
  <section className={classNames("af-timeline-vertical", className)}>
    <header className="af-timeline-vertical__header">
      {tag}
      <h4 className="af-timeline-vertical__title">{title}</h4>
    </header>
    {Boolean(children) && (
      <main className="af-timeline-vertical__description">{children}</main>
    )}
  </section>
);
```